docs/source/app-dev/command-deduplication.rst

```diff
@@ -85,6 +85,8 @@ Independently of how the outcome is communicated, command deduplication generate
   Neither deduplication durations up to the :ref:`maximum deduplication time <com.daml.ledger.api.v1.LedgerConfiguration.max_deduplication_time>` nor deduplication offsets published within that time SHOULD result in this error.
   Participants may accept longer periods at their discretion.
 
+- The gRPC status code ``FAILED_PRECONDITION`` with error code id :ref:`PARTICIPANT_PRUNED_DATA_ACCESSED <error_code_PARTICIPANT_PRUNED_DATA_ACCESSED>`, when specifying a deduplication period represented by an offset, indicates that the specified deduplication offset has been pruned.
+  The field ``earliest_offset`` in the metadata specifies the last pruned offset.
```

Comment:
Reading it here suggests to me that this is (or will be soon, i.e. before 2.0) supported by all ledgers and and we do allow applications to rely upon this info. Is this correct and is it intended?

Reply:
Yes as this is basically part of the participant therefore it's not KV specific.